apps/internal/local/server_test.go

```diff
@@ -129,6 +217,61 @@ func TestServer(t *testing.T) {
 			continue
 		}
 
+		if len(test.successPage) > 0 {
+			if !bytes.Equal(content, test.successPage) {
+				t.Errorf("TestServer(%s): -want/+got:\ntest option error page", test.desc)
+			}
+			continue
+		}
+
+		if test.testTemplate {
+			if test.testErrCodeXSS || test.testErrDescriptionXSS {
+				if !strings.Contains(string(content), "&lt;script&gt;alert(&#39;this code snippet was executed&#39;)&lt;/script&gt;") {
```

Comment:
It would be much simpler to add the (hardcoded) expected output to `test`. That would let you write e.g. `if !bytes.Equal(test.expected, content)` instead of all these conditions